### PR TITLE
Feature/wam hackaton ci and google chrome csv import

### PR DESCRIPTION
**Pull request by gitcoindev**
Hi, this is my submission for WAM hackathon, I described the pull request at https://hackmd.io/@en6I0FFLT6iJvOcdkCuOXg/SkgEOqlGc . Thank you for reviewing it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # FairPass
 
+[![Build](https://github.com/gitcoindev/FairPass/workflows/Build/badge.svg)](https://github.com/gitcoindev/FairPass/actions?query=workflow%3ABuild) [![Test](https://github.com/gitcoindev/FairPass/workflows/Test/badge.svg)](https://github.com/gitcoindev/FairPass/actions?query=workflow%3ATest) [![Static code analysis](https://github.com/gitcoindev/FairPass/workflows/Static%20code%20analysis/badge.svg)](https://github.com/gitcoindev/FairPass/actions?query=workflow%3AStatic%20code%20analysis)
```
Comment (cryptomental) [inline README.md:3]: I noticed the badges point to my fork, after merging the github.com/gitcoindev path will change to github.com/fairDataSociety, apart from that no changes will be needed.

**Comment by gitcoindev** [comment]
Closing the pull request as won't merge. There was no communication and will to review for the last 10 days.

**Comment by crtahlin** [comment]
Hi @gitcoindev 

Thank you for your participation at WAM and your work you did on this issue. 

Although it didn't meet the complete requirement list, the parts you did would be very useful and supportive for the FairOS  Password manager workflow. 

If you would be interested, we can also continue on the FDS Discord (sent you a friend request, I hope I have the right user).

**Comment by gitcoindev** [comment]
Hi @crtahlin , @asabya I am back. Please let me know which parts of the pull request to remove or rework. Let's work on merging this ! -)

**Comment by asabya** [comment]
Hello @gitcoindev, The changes in the PR can be easily merged. Just wanted to ask one thing, is there any specific reason that you didn't use [matrix](https://docs.github.com/en/actions/using-jobs/using-a-build-matrix-for-your-jobs) for the build process on different operating system?

**Comment by gitcoindev** [comment]
Hi @asabya, there was no specific reason. I can try to refactor the build process to use matrix, will let you know when it's completed. Should be ready today, tomorrow at the latest.

**Comment by gitcoindev** [comment]
Hi @asabya I refactored the build process to use GitHub matrix and I also added release configuration with GitHub matrix support. An example is available at https://github.com/gitcoindev/FairPass/releases/tag/v0.0.8 and the logs at https://github.com/gitcoindev/FairPass/actions/runs/2196043913 .

**Review by asabya — APPROVED** [review]
LGTM. @tfius @crtahlin should we merge this?

**Comment by tfius** [comment]
> LGTM. @tfius @crtahlin should we merge this?

yes

**Pull request merged**